.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,8 +20,10 @@ Section: New features, Changes, Bug fixes
 ### Code Review Checklist:
 
 This checklist is a reminder of things commonly forgotten in a new PR.
-Please do a self-review to check these items.
-Reviewers will not approve the PR until these are met.
+Authors, please do a self-review and confirm you have considered the following items.
+Mark items you have considered by checking them.
+You can do this when editing the Pull request description with an x: `[ ]` becomes `[x]`.
+You can do also check the checkboxes after the PR is created.
```

Comment:
This sentence sounds strange to me. Is there any extra "do" in it?
Feel free to ignore my suggestion if the original is correct English (using "do" to insist), I am not a native English speaker.

```suggestion
You can also check the checkboxes after the PR is created.
```

Reply:
Yes you are right. Good catch.